tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -55,6 +58,21 @@ if(WIN32)
   )
 endif()
 
+add_custom_command(
+  TARGET Engine_Tests POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E copy_directory
+  $<TARGET_FILE_DIR:DoTest>
+  $<TARGET_FILE_DIR:Engine_Tests>/Resources/Plugins
+)
+
+#TODO: This is REALLY terrible but whatever, don't do this at home kids
```

Comment:
Wasn't there a fix we had  for this? Not sure if this is still necessary or not

Reply:
not afaik?

Reply:
I double checked and I don't see it either - must've been dreaming :^)